internal/clients/diffreviewer/github/github_service.go

```diff
@@ -62,38 +63,22 @@ type headCommit struct {
 
 // event represents github event webhook file
 type event struct {
+	Before      string      `json:"before"`
+	BaseRef     string      `json:"base_ref"`
```

Comment:
is this the field that doesn't get populated?

Reply:
Yep, will remove!